executor-scripts/linux/batman

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# Maximilian Wilhelm <max@sdn.clinic>
+#  --  Wed 26 Aug 2020 08:15:58 PM CEST
+#
+# This executor is responsible for setting up the main B.A.T.M.A.N. adv. interface (eg. bat0)
+# as well as managing settings of the underlying interfaces (hardifs).
+#
+# Know options for the main interface are:
+#  IF_BATMAN_IFACES			Space separated list of interfaces to be part of this B.A.T.M.A.N. adv. instance
+#  IF_BATMAN_IFACES_IGNORE_REGEX	Interfaces to ignore when verifying configuration (regexp)
+#  IF_BATMAN_DISTRIBUTED_ARP_TABLE	'enable' or 'disable' DAT of this B.A.T.M.A.N. adv. instance
+#  IF_BATMAN_MULTICAST_MODE		'enable' or 'disable' the multicast mode of this B.A.T.M.A.N. adv. instance
+#  IF_BATMAN_GW_MODE			Set the gateway mode of this B.A.T.M.A.N. adv. instance
+#  IF_BATMAN_HOP_PENALTY		Set the hop penalty of this B.A.T.M.A.N. adv. instance
+#
+# Known options for underlying interfaces are:
+#  IF_BATMAN_IFACE_PENALTY		Set the hop penalty of this B.A.T.M.A.N. adv. interface
+#
+set -e
+
+if [ "$VERBOSE" ]; then
+	set -x
+fi
+
+if [ "${IF_BATMAN_IFACES}" -o "${IF_BATMAN_IFACE_PENALTY}" ]; then
+	if ! which batctl >/dev/null 2>&1; then
+		echo "Error: batctl utility not found!" >&2
+		exit 1
+	fi
+fi
+
+#
+# Compatiblity glue: Newer versions of batctl print a deprecation
+# warning when called with -m <batif>. Avoid spamming the log and
+# producting SPAM by silently handling this here.
+mesh_if_param="-m"
+if batctl -h 2>&1 | grep -q "meshif"; then
+        mesh_if_param="meshif"
+fi
+
+#
+# Functions to manage main B.A.T.M.A.N. adv. interface
+batctl_if () {
+	for iface in ${IF_BATMAN_IFACES}; do
+		${MOCK} batctl "${mesh_if_param}" "${IFACE}" interface "$1" "${iface}"
+	done
+}
+
+set_batmam_params () {
```

Comment:
shouldn't this be `set_batman_params()` ?

Reply:
Hm, it works with /bin/sh being dash on Debian I think. Am I violating POSIX? :)